fhir-persistence/src/test/resources/logging.unitTest.properties

```diff
@@ -49,5 +49,5 @@ java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
 ############################################################
 
 # Set this to FINE or higher to output the SQL statements and related debug info
-com.ibm.fhir.persistence.level = INFO
-com.ibm.fhir.search.level = INFO
+com.ibm.fhir.persistence.level = FINE
+com.ibm.fhir.search.level = FINE
```

Comment:
probably want to flip these back to INFO

Reply:
fixed